stylesheets/tiny-slider/tiny-slider.scss

```diff
@@ -1,140 +1,173 @@
 .tns-outer {
   padding: 0 !important;
 }
+
 .tns-outer [hidden] {
   display: none !important;
 }
+
 .tns-outer [aria-controls],
 .tns-outer [data-action] {
   cursor: pointer;
 }
+
 .tns-slider {
-  -webkit-transition: all 0s;
-  -moz-transition: all 0s;
+  transition: all 0s;
+  transition: all 0s;
```

Comment:
you'd think stylelint would be smarter and yet

Reply:
🤦 
```suggestion
  transition: all 0s;
```